operations/commit_queue.go

```diff
@@ -636,6 +636,8 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient, u
 	if err = patchParams.validateSubmission(diffData); err != nil {
 		return err
 	}
+	//here we go. test here tomorrow and compare it with the regular patch command
```

Comment:
we should remove this i imagine?

Reply:
LOL my bad. I'm glad it wasn't anything more embarrassing than this.